README.md

```diff
@@ -25,3 +30,77 @@ Home Assistant component that interfaces FEMS and OpenEMS, mainly used in Feneco
 2. Manually add this repository to HACS
 4. Add FEMS Integration
 6. Enter your OpenEMS / FEMS address and user account (Fenecon standard:  x / user)
+
+The Fenecon password (user or owner) controls your access rights. If you just want to monitor (read-only access), "user" is fine. If you want to change settings, you need to use "owner".
+
+## Installation
+
+After installation most devices and entities are disabled. You can enable them like this:
+
+<img src="https://github.com/user-attachments/assets/3c4619fa-41a7-4b20-a4f5-46b1fd692cfc" width="750">
+
+
+Some devices to watch for:
+
+1. ctrlIoHeatPump0
+2. meter0
+3. timeOfUseTariff0
+4. batteryInverter0
+5. charger0 and charger1 (these are inverter strings)
+
+## Entites
+
+### _ssum
+The most relevant entities (for most people) are in the _sum device:
+
+<img src="https://github.com/user-attachments/assets/95507715-3e03-43f3-ae13-e46cf0ffc5e2" width="200">
+
+Here you can find your power production, consumption, battery charge and many more.
+
+### Charger0 and 1
```

Comment:
I'd make this more generic. String count and number depends on the configuration, for my strings it's `charger10` and `charger11` - I'd just change it to something like `chargerN` or `chargerX` or whatever to indicate this depends on the setup.